client/me/purchases/concierge-banner/index.jsx

```diff
@@ -21,7 +21,7 @@ class ConciergeBanner extends Component {
 				<ActionCard
 					headerText={ this.props.translate( 'Looking for Expert Help?' ) }
 					mainText={ this.props.translate(
-						'Get 30 minutes dedicated to the success of your site. Schedule your free 1-1 concierge session with a Happiness Engineer!'
+						'Get 30 minutes dedicated to the success of your site. Schedule your free 1-1 Quick Start session with a Happiness Engineer!'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Get 30 minutes dedicated to the success of your site. Schedule your free 1-1 Quick Start Session with a Happiness Engineer!' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-features-taglines)